server/service/middleware/ratelimit/ratelimit.go

```diff
@@ -98,7 +111,7 @@ func (m *ErrorMiddleware) Limit(keyName string, quota throttled.RateQuota) endpo
 			if err != nil {
 				_, _, rateErr := limiter.RateLimit(ipKeyName, 1)
 				if rateErr != nil {
-					return nil, ctxerr.Wrap(ctx, err, "check rate limit")
+					return nil, ctxerr.Wrap(ctx, err, "failed to rate limit")
```

Comment:
All these changes return the same error message. I like to make the error messages unique since it makes it easier to debug if I know immediately where the error is coming from.

Reply:
Good idea!

Reply:
[Make error strings unique](https://github.com/fleetdm/fleet/pull/16334/commits/865c351016b480d32b5b1e64a76584cb13dcf12d)